cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -6,9 +6,10 @@ from dynamic_reconfigure.parameter_generator_catkin import *
 gen = ParameterGenerator()
 
 damping_method_enum = gen.enum([ 
-                       gen.const("No_damping",      int_t, 0, "No damping means a damping factor of 0.0."),
-                       gen.const("Constant",        int_t, 1, "Constant damping factor given by parameter damping_factor."), 
-                       gen.const("Manipulability",  int_t, 2, "Damping factor calculated by a function of manipulability measure.")], 
+                       gen.const("No_damping",           int_t, 0, "No damping means a damping factor of 0.0."),
+                       gen.const("Constant",             int_t, 1, "Constant damping factor given by parameter damping_factor."), 
+                       gen.const("Manipulability",       int_t, 2, "Damping factor calculated by a function of manipulability measure."), 
+                       gen.const("Low_Isotropic_Damping",int_t, 3, "Damping factor is calculated dynamically between 0 and lambda_max")],
```

Comment:
This is what I understood about LowIsotropicDamping:
It is __not__ a full (stand-alone) damping_method on its own but the part ```... + beta*Identitymatrix + ...``` in the NumericalFiltering equation ```q_dot = J^#*X_dot```, with ```J^# = J_eT*(Je*JeT + beta*Identitymatrix + lambda*(u_m*u_mT))^-1```. Isn't this so? See (20) in Chiaverini: Singularity-robust task-priority redundancy resolution

Thus, please remove ```Low_Isotropic_Damping``` from the damping_methods!
Rather add a new damping_method ```SmallestSingularValue``` where ```get_damping_factor()``` returns a value as given in (11.23) of Siciliano: Chapter 11 

Further, I still think that ```ǸumericalFiltering``` should be another new damping_method.
In order to achieve this, I think DampingBase should be extended in a way that it does not return a value for ```lambda``` but return a value for ```F(lambda)``` where ```F(lambda) = lambda*Identity``` for most damping_methods and ```F(lambda)=beta*Identity + lambda*(u_m*u_mT)``` for ```NumericalFiltering```

What do you think?